src/imgui/DebuggableEditor.cc

```diff
@@ -17,6 +19,7 @@
 #include "unreachable.hh"
 
 #include "imgui_stdlib.h"
+#include "imgui_internal.h"
```

Comment:
Detail: now that we've introduced the `checkShortcut()` helper function, fewer files will need to include this experimental header file.